R/package.R

```diff
@@ -168,12 +164,9 @@ check_bootstrap_version <- function(version, pkg) {
     cli::cli_warn("{.var bootstrap: 4} no longer supported, using {.var bootstrap: 5} instead")
     5
   } else {
-    msg_fld <- pkgdown_field(pkg, c("template", "bootstrap"), cfg = TRUE, fmt = TRUE)
-    cli::cli_abort(
-      c(
-        "Boostrap version must be 3 or 5.",
-        x = paste0("You set a value of {.val {version}} to ", msg_fld, ".")
-      ),
+    config_abort(
+      pkg,
+      "{.field template.bootstrap} must be 3 or 5, not {.val {version}}.",
```

Comment:
would it make sense for `config_abort` to take a help topic/URL as optional argument, so that the error messages could point to an useful page? Here it would be the link to the article about the customization.

Reply:
IMO, it's not that useful in general, especially for errors that I think will be relatively uncommon (like this one). We can alway add as a bullet where needed (see above).